docs/en-us/architecture.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Architecture - Apache TubeMQ
+---
```

Comment:
![image](https://user-images.githubusercontent.com/16833559/83991431-4b732700-a97f-11ea-927e-e042671f040d.png)

We already have a title "TubeMQ Architecture". Why add a new title?

Reply:
Thanks @guangxuCheng  for the comment.

This title is a metadata item, it is used in DocSite generation page (from markdown .md file), it is not displayed on page (html).
```
For markdown documents, docsite provides three metadata configuration items:
title, keywords, and description. At the same time, the corresponding jsx page 
is rendered into static HTML and combined into the final generated page.
```
Ref. https://docsite.js.org/en-us/docs/SEO.html

Reply:
got that.

Reply:
i think the title should not include ```Apache TubeMq```,it may be a trouble to remind all docs to add the title

Reply:
@wf123537200 Do you mean the title doesn't need to include `Apache TubeMQ` suffix?

Reply:
If adding title metadata to the document is cumbersome, an even more extreme approach is to modify the template file to define the title tag directly as a specified string like "Apache TubeMQ".

I think that would solve this issue as well, but I don't think it's appropriate considering the search engine optimization.

What do you guys think?

Reply:
Metadata title is docsite's syntax not markdown, it is easy for contributors or reviewers to ignore this title when adding new document which will lead to inconsistent document formats on website. In addition, whether or not to add this title will not affect the user's reading. 

I just spoke out my worries, anyway, +1 for the pr